frontend/main.js

```diff
@@ -434,6 +434,8 @@ async function startApp () {
         }
         this.ephemeral.finishedLogin = 'yes'
 
+        sbp('gi.actions/identity/kv/load')
```

Comment:
All errors need to handled some way so that users can send us useful error logs if there are problems.

So here we need something like:

```js
.catch(e => {
  console.error("Error from 'gi.actions/identity/kv/load' during login:", e)
})
```